argus/tests/cloud/smoke.py

```diff
@@ -286,7 +286,9 @@ class TestPowershellMultipartX86TxtExists(base.BaseTestCase):
 
     def test_file_exists(self):
```

Comment:
you need to add a new test specifically for the userdatafile and skip it on nano